src/core/webview/generateSystemPrompt.ts

```diff
@@ -82,6 +83,7 @@ export const generateSystemPrompt = async (provider: ClineProvider, message: Web
 		maxReadFileLine !== -1,
 		{
 			maxConcurrentFileReads,
+			disabledTools,
 		},
```

Comment:
packages/types/src/tool.ts this is the single point of truth for all tools: because this list is updated with new tools get added the PR needs to be dynamic and based on this list.